Tutorial_1a_Convolving_Images.ipynb

```diff
@@ -0,0 +1,650 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from astropy.convolution import convolve
</pre><p>Also could note the <code>convolve_fft</code> could also be used (and is usually faster for small kernel sizes) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>